alica_engine/include/engine/PlanRepository.h

```diff
@@ -42,59 +43,100 @@ class PlanRepository {
     PlanRepository();
```

Comment:
https://msdn.microsoft.com/en-us/library/dn457344.aspx it seems mentioning constructor as default and removing the empty constructor definition makes it more efficient

Reply:
That is not universally true. Putting default(=empty) constructors (and destructors) in the cpp file can actually speed things up.